frontend/src/views/student/submissions/EditSubmissionDialog.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-dialog
+    :value="dialog"
+    @input="$emit('dialog', false)"
+    @keydown.esc="$emit('dialog', false)"
+    max-width="75%"
+    max-height="80%"
+  >
+    <v-card>
+      <v-card-title>
+        <span class="headline">
+          {{ editMode(editSubmission) ? 'Edit Submission' : 'New Submission' }}
+        </span>
+      </v-card-title>
+
+      <v-card-text class="text-left" v-if="editSubmission">
+        <v-text-field
```

Comment:
It would make sense to reuse a component for question, specially when in the future there will be several types of questions. See how is the code of the pull request on questions